db_stress_tool/db_stress_test_base.cc

```diff
@@ -1230,26 +1255,30 @@ void StressTest::OperateDb(ThreadState* thread) {
           }
         }
 
-        if (total_size <= FLAGS_backup_max_size && fault_fs_guard) {
+        if (total_size <= FLAGS_backup_max_size) {
           // TODO(hx235): enable error injection with
           // backup/restore after fixing the various issues it surfaces
-          fault_fs_guard->DisableThreadLocalErrorInjection(
-              FaultInjectionIOType::kMetadataRead);
-          fault_fs_guard->DisableThreadLocalErrorInjection(
-              FaultInjectionIOType::kMetadataWrite);
-          fault_fs_guard->DisableThreadLocalErrorInjection(
-              FaultInjectionIOType::kRead);
-          fault_fs_guard->DisableThreadLocalErrorInjection(
-              FaultInjectionIOType::kWrite);
+          if (fault_fs_guard) {
```

Comment:
Just for a thought (Not about this PR) 

I see we do this in a few different places. Just wondering if it's worth creating a macro for this for convenience. (e.g. if we add a new FaultInjectionIOType, we only have to add this to a single place)

Reply:
yeah could be a good follow up.